src/hotspot/share/jfr/jfr.hpp

```diff
@@ -28,6 +28,11 @@
 #include "jni.h"
 #include "memory/allocation.hpp"
 
+#include "jfr/support/jfrIntrinsics.hpp"
+#ifdef JFR_HAVE_INTRINSICS
+#include "jfr/recorder/checkpoint/types/traceid/jfrTraceIdEpoch.hpp"
```

Comment:
Please create a Jfr::epoch_address() in the .hpp and move includes to the .cpp - thanks.

Reply:
fixed.